src/Http/Response.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+namespace Framework3\Http;
+
+use Framework3\Bag\Bag;
+use Framework3\Http\Session\Session;
+use Framework3\Http\Session\SessionInterface;
+
+class Response
+{
+    public const STATUS_OK =            ['code' => 200, 'name' => '200 OK'];
```

Comment:
Codacy found an issue: [Equals sign not aligned with surrounding assignments; expected 11 spaces but found 1 space](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6299285)